onnx/defs/rnn/defs.cc

```diff
@@ -90,15 +90,22 @@ std::function<void(OpSchema&)> RNNDocGenerator(const char* /*name*/) {
         "X",
         "The input sequences packed (and potentially padded) into one 3-D "
         "tensor with the shape of `[seq_length, batch_size, input_size]`.",
-        "T");
+        "T",
+        OpSchema::Single,
+        true,
+        1,
+        OpSchema::Differentiable);
     schema.Input(
         4,
         "sequence_lens",
         "Optional tensor specifying lengths of the sequences in a batch. "
         "If not specified - assumed all sequences in the batch to have "
         "length `seq_length`. It has shape `[batch_size]`.",
         "T1",
-        OpSchema::Optional);
+        OpSchema::Single,
```

Comment:
```suggestion
        OpSchema::Optional,
```
This should be `OpSchema::Optional`. All `optional` variables need to remain `optional`.